flixel/input/gamepad/FlxGamepadManager.hx

```diff
@@ -549,4 +657,11 @@ class FlxGamepadManager implements IFlxInputManager
 		}
 		return globalDeadZone;
 	}
+	
+	#if (!flash && !next)
+	
+	private static inline var JOYSTICK_BUTTON_UP: "buttonUp";
```

Comment:
Nice, this even added some self-documentation because of the added `JOYSTICK`. :+1: